### PR TITLE
gstreamer: 1.12.2 -> 1.12.3

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by LnL7** [comment]
@GrahamcOfBorg eval

**Comment by orivej** [comment]
Merged into staging for better testing.

File: pkgs/development/libraries/gstreamer/validate/default.nix [hunk 1]
```diff
@@ -3,7 +3,7 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "gst-validate-1.12.2";
+  name = "gst-validate-1.12.3";
```
Comment (vcunat) [inline pkgs/development/libraries/gstreamer/validate/default.nix]: Forgot to update the hash (and test apparently).

Comment (vcunat) [reply pkgs/development/libraries/gstreamer/validate/default.nix]: The same holds also for -vaapi, -python, -editing-services.

Comment (vcunat) [reply pkgs/development/libraries/gstreamer/validate/default.nix]: Note: due to openssl security update, this now got to master, so any fixes should go there directly.

Comment (orivej) [reply pkgs/development/libraries/gstreamer/validate/default.nix]: Fixed in 9ebcc8db01cd1797a8bfcdd5b04643e6e92b6bb2

**Comment by magnetophon** [comment]
@vcunat @orivej Thanks, and sorry for the sloppiness!

**Pull request merged**